src/sentry/services/hybrid_cloud/organization_mapping/serial.py

```diff
@@ -22,4 +20,12 @@ def update_organization_mapping_from_instance(
 
 
 def serialize_organization_mapping(org_mapping: OrganizationMapping) -> RpcOrganizationMapping:
-    return cast(RpcOrganizationMapping, RpcOrganizationMapping.serialize_by_field_name(org_mapping))
+    return RpcOrganizationMapping(
+        id=org_mapping.organization_id,
+        slug=org_mapping.slug,
+        name=org_mapping.name,
+        region_name=org_mapping.region_name,
+        date_created=org_mapping.date_created,
+        verified=org_mapping.verified,
```

Comment:
Aren't we removing the `verified` attribute soon @GabeVillalobos ?

Reply:
We are indeed. Right now we're always defining it as false per the model default and shouldn't be setting it anywhere.

Reply:
From the mapping you mean? So there's no harm in just proactively removing it from the serialize function here?

Reply:
Yes, organization mapping will no longer have a `verified` property or column in the database very soon. Removing it proactively from here should be fine.